sstvis.py

```diff
@@ -215,27 +217,31 @@ def process_line(self, line):
             self.triangle_count += 1
 
 
-if __name__ == "__main__":
+@click.command()
+@click.option('--thinning', default=THINNING_FACTOR, help='Number of greetings.')
+@click.option('--frequency', default=UPDATE_FREQUENCY, help='The person to greet.')
```

Comment:
@hugoledoux I think the `help` args are wrong here 😇

Reply:
you think?